src/Microsoft.Health.Checkpoints/Checkpoint.cs

```diff
@@ -0,0 +1,22 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+
+namespace Microsoft.Health.Checkpoints
+{
+    public class Checkpoint : ICheckpoint
+    {
+        public string Partition { get; set; }
+
+        public string Identifier { get; set; }
+
+        public Azure.ETag ETag { get; set; }
```

Comment:
I don't know if we need to have this as the Azure.ETag object.  Can we simplify the interface and represent this as a string and interpolate between the two internally?

Reply:
yep, updated to string.